### DISCUSSION TITLE
Range of values for latent variables?

### DISCUSSION THREAD

**Discussion by gerverska** [JenniNiku/gllvm#185] [Q&A] [answered]
Hi folks,

Fitting a binomial model, I noticed that the range of values for LV1 and LV2 were a lot smaller (each from ~ -5e-7 to 5e-7) than what I've observed for other types of models (-2 to 2).

Does the range of values for an LV matter? Or do the values only offer interpretation within the context of the model used to produce the LV ordination?

**Comment by BertvanderVeen** [comment, marked as answer]
The range matters in terms of how much an LV affects the response. You can try to change method to EVA or LA to double check if you get the same issue, but generally speaking, binomial GLLVMs take on small values for LVs either when 1) there is insufficient information to estimate the LV, or 2) the LV is redundant.

**Reply by gerverska** [reply]
Thanks, Bert. So if I do have:

- two LVOs
- each modeling different types of responses (e.g., presence-absence vs count)
- but the structure of the sample x species matrix is the same

then a smaller range of values for the LVs of the presence-absence model suggests less info/greater redundancy relative to the count model, right?

On one hand, wouldn't we nearly always expect presence-absence models to have less information than count models? Are orders of magnitude differences between LVs of two different LVOs inherently suspect? Because it seems this outcome might just be a reflection of real properties of the data.

**Reply by gerverska** [reply]
Just to provide more context for this discussion: Bert's suggestion to examine LA / EVA for the binomial (with probit) was very helpful. A presence-absence dataset I have been looking at recently, when fit with `binomial('probit')` + `method = 'VA'` was giving me an LV associated with very little variance (~ 1e-16). This created issues for calculating a pseudo-R2:
```
# From Jenni's first vignette ####
rcov <- getResidualCov(fit_env)
rcov0 <- getResidualCov(fit1lv)
1 - rcov$trace / rcov0$trace
```
Even though my full model + LV had a substantially lower BIC (using this because I have a large sample size) than the null model (only LV), the traces for each model essentially became the same after `adjust = 1` was applied (i.e., converging on the number of taxa I have).

Switching to `method = LA`, I get values of ~100 that actually vary in expected ways between full and null models. I'm not sure why VA vs LA are so divergent here, but I'm inclined to trust the LA given what I know about my data (examining plots of taxa vs explanatory variables, etc.).

**Reply by BertvanderVeen** [reply]
Good! There is a know issue with the VA Probit model that I think might be affecting your model - hence my suggestion.
